### PR TITLE
Improve haproxy 

### PR DESCRIPTION
**Pull request by ktarasz**
Added:
- hrsp_1xx, hrsp_2xx, hrsp_3xx, hrsp_4xx, hrsp_5xx, hrsp_other, hrsp_total for backands and frontends 
- qtime, ctime, rtime, ttime metrics for backand servers
- Backend Servers In UP State

### CODE REVIEW

**Comment by l2isbad** [comment]
gj @ktarasz 👍 
Please post a few screenshots with the new charts

File: python.d/haproxy.chart.py [hunk 1]
```diff
@@ -39,6 +44,34 @@
         'options': [None, "Session In Queue", "sessions", 'frontend', 'haproxy_f.qcur', 'line'],
         'lines': [
         ]},
+    'fhrsp_1xx': {
+        'options': [None, "HTTP responses with 1xx code", "responses", 'frontend', 'haproxy_f.hrsp_1xx', 'line'],
```
Comment (l2isbad) [inline python.d/haproxy.chart.py]: I think all units should be `responses/s` not `responses` since algorithm is incremental (true for all new charts)

**Comment by sebbrandt87** [comment]
would it help to merge it if I paste screenshots from charts?

**Comment by l2isbad** [comment]
@sebbrandt87 
The only problem i see is units (`responses`, should be `responses/s`).
Not a big deal. In general i am ok to merge it @ktsaou

**Comment by ktsaou** [comment]
ok, perfect!

@ktarasz please fix the unit names and let me know when you are ready, to merge this.

**Comment by ktsaou** [comment]
merged! thank you!

**Pull request merged**